docs/topics/plugin-intro.adoc

```diff
@@ -9,5 +9,5 @@
 This guide is for engineers, consultants, and others who want to use the {PluginName} for the {ProductName} ({ProductShortName}) to assist with migrating applications.
 
 ifndef::vscode-plugin-guide[]
-NOTE: This guide uses _Eclipse_ to refer to an installation of Eclipse or Red Hat CodeReady Studio.
+// NOTE: This guide uses _Eclipse_ to refer to an installation of Eclipse.
```

Comment:
I see that this line is commented out, which is good. Maybe it can be deleted altogether?

Reply:
@anarnold97 As far as i can tell.